guides/common/modules/proc_obfuscating-hostnames-and-ip-addresses.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="obfuscating-hostnames-and-ip-addresses"]
+= Obfuscating hostnames and IP addresses
+
+You can use {ProjectName} to obfuscate hostnames and IP addresses when uploading host data to {Insights}. 
+Obfuscation ensures that sensitive identifying information is not exposed. 
+When obfuscation settings are enabled, {Project} obfuscates hostnames and IP addresses. 
+If obfuscation settings are enabled and the host has its own `insights-client` obfuscation settings, {Project} honors the host settings.  
+If obfuscation settings are disabled, {Project} does not obfuscate hostnames or IP addresses, regardless of client settings.
+
+By default, {Project} applies global obfuscation settings unless an `insights-client` on the host overrides the settings with its own configuration.
+This procedure describes how {Project} applies obfuscation settings.
+
+.Prerequisites
+* Your hosts have `insights-client` installed and configured.
+
+.Procedure
+. Navigate to *{Insights}* > *Inventory Upload*.
+. Enable the *Obfuscate host names* and *Obfuscate host ipv4 addresses* settings to obfuscate host names and ipv4 addresses sent to the {RHCloud}.
+
+.Verifying obfuscation
```

Comment:
```suggestion
.Verification
```
We have to stick to the modularization structure strictly. "Verification" is an allowed discrete heading in [procedure modules](https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs) whereas "Verifying obfuscation" is not.